src/xunit.analyzers/AssertEqualLiteralValueShouldBeFirstFixer.cs

```diff
@@ -34,10 +35,23 @@ public class AssertEqualLiteralValueShouldBeFirstFixer : CodeFixProvider
         private async Task<Document> SwapArgumentsAsync(Document document, InvocationExpressionSyntax invocation, CancellationToken cancellationToken)
         {
             var editor = await DocumentEditor.CreateAsync(document, cancellationToken).ConfigureAwait(false);
-            var firstArg = invocation.ArgumentList.Arguments[0];
-            var secondArg = invocation.ArgumentList.Arguments[1];
-            editor.RemoveNode(firstArg);
-            editor.InsertAfter(secondArg, firstArg);
+
+            var arguments = invocation.ArgumentList.Arguments;
+            if (arguments.All(x => x.NameColon != null))
```

Comment:
C# 7.2 allows the omission of argument names as long as they are positionally in the correct place, e.g. `Assert.Equal(expected: i, 123)`. Certainly an unlikely edge case, but could you make sure that this new syntax is supported?

Reply:
I actually consider the new syntax, but found out that the order cannot be changed with this, so I thought it wasn't applicable. This is a good scenario, I've covered it now, as well.